services/api/config.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    """config for api"""
+    SQLALCHEMY_DATABASE_URL: str
```

Comment:
maybe POSTGRES_URL is a better name? ie something that specifies what it connects to...though prob doesn't matter too much here